tests/lib/SimpleSAML/XML/SignerTest.php

```diff
@@ -78,12 +83,14 @@ public function testSignerBasic()
      */
     public function testSignBasic()
     {
-        $node = new \DOMDocument();
+        $node = DOMDocumentFactory::create();
```

Comment:
I don't think it's necessary to do this. These are just tests and we control the XML, so there's no point on adding an extra level of indirection just to create a `DOMDocument` object.

Reply:
Reverted